### PR TITLE
Create zh_cn.json

### PR DESCRIPTION
**Pull request by qsefthuopq**

### CODE REVIEW

File: src/main/resources/assets/ltr/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "text.ltr.meditation2": "现在，想象一下能量之球环绕着你。伸出你的手收集它们。将它们带到柱子处...",
+    "text.ltr.meditation3": "随着你搭建柱子，你会发现你被不和谐的力量所包围，这些力量阻止着你悟道...",
+    "text.ltr.meditation4": "有时候，这股不和谐的力量动摇了你的柱子...",
+    "text.ltr.meditation5": "当你最需要它的时候，整体的光线就会很明亮。沐浴在它的光辉中，直到你感到它为你的柱子注入了活力...",
+    "text.ltr.meditation6": "透过你的心灵之眼来审视你自己。像别人看你一样审视自己。从短距离的各个角度审视自己...",
+    "text.ltr.meditation7": "想象一下，你正漂浮在一片纯粹的祥云中。如果云是淡紫色的，吟诵一句平静的话，直到它没有...",
+    "text.ltr.meditation8": "放空你的大脑。专注于柱子，专注于驱散不和谐的力量...",
+    "text.ltr.meditation9": "你的超验悟道之旅可能很长，但耐心是你维持精神健康的关键...",
+    "text.ltr.do_it": "行动吧！",
+    "subtitle.ltr.hwndu": "Honeyboy说道",
```
Comment (liach) [inline src/main/resources/assets/ltr/lang/zh_cn.json]: https://en.wikipedia.org/wiki/Honey_Boy_(film)
```suggestion
    "subtitle.ltr.hwndu": "宝贝男孩说道",
```

File: src/main/resources/assets/ltr/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "itemGroup.ltr.core_group": "土豆仔",
+
+    "block.ltr.lil_tater": "土豆仔",
+
+    "text.ltr.mySon": "别再和我或",
+    "text.ltr.mySon_single": "我的儿子说话。",
+    "text.ltr.mySon_multiple": "我的儿子说话。",
+    "text.ltr.imitater_lil": "仔",
+    "text.ltr.meditation0": "深呼吸。吸气... 2，3... 屏息, 2，3... 呼气。 2，3，4... 不错。再来一次...",
```
Comment (liach) [inline src/main/resources/assets/ltr/lang/zh_cn.json]:
```suggestion
    "text.ltr.meditation0": "深呼吸。吸气…2，3…屏息, 2，3…呼气。 2，3，4…不错。再来一次…",
```

File: src/main/resources/assets/ltr/lang/zh_cn.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "text.ltr.meditation1": "想象你的悟道之路是三根直入云霄的柱子...",
```
Comment (liach) [inline src/main/resources/assets/ltr/lang/zh_cn.json]:
```suggestion
    "text.ltr.meditation1": "想象你的悟道之路是三根直入云霄的柱子…",
```

File: src/main/resources/assets/ltr/lang/zh_cn.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    "text.ltr.meditation2": "现在，想象一下能量之球环绕着你。伸出你的手收集它们。将它们带到柱子处...",
```
Comment (liach) [inline src/main/resources/assets/ltr/lang/zh_cn.json]:
```suggestion
    "text.ltr.meditation2": "现在，想象一下能量之球环绕着你。伸出你的手收集它们。将它们带到柱子处…",
```

File: src/main/resources/assets/ltr/lang/zh_cn.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    "text.ltr.meditation3": "随着你搭建柱子，你会发现你被不和谐的力量所包围，这些力量阻止着你悟道...",
```
Comment (liach) [inline src/main/resources/assets/ltr/lang/zh_cn.json]:
```suggestion
    "text.ltr.meditation3": "随着你搭建柱子，你会发现你被不和谐的力量所包围，这些力量阻止着你悟道…",
```

File: src/main/resources/assets/ltr/lang/zh_cn.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    "text.ltr.meditation4": "有时候，这股不和谐的力量动摇了你的柱子...",
```
Comment (liach) [inline src/main/resources/assets/ltr/lang/zh_cn.json]:
```suggestion
    "text.ltr.meditation4": "有时候，这股不和谐的力量动摇了你的柱子…",
```

File: src/main/resources/assets/ltr/lang/zh_cn.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    "text.ltr.meditation5": "当你最需要它的时候，整体的光线就会很明亮。沐浴在它的光辉中，直到你感到它为你的柱子注入了活力...",
```
Comment (liach) [inline src/main/resources/assets/ltr/lang/zh_cn.json]:
```suggestion
    "text.ltr.meditation5": "当你最需要它的时候，整体的光线就会很明亮。沐浴在它的光辉中，直到你感到它为你的柱子注入了活力…",
```

File: src/main/resources/assets/ltr/lang/zh_cn.json [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    "text.ltr.meditation6": "透过你的心灵之眼来审视你自己。像别人看你一样审视自己。从短距离的各个角度审视自己...",
```
Comment (liach) [inline src/main/resources/assets/ltr/lang/zh_cn.json]:
```suggestion
    "text.ltr.meditation6": "透过你的心灵之眼来审视你自己。像别人看你一样审视自己。从短距离的各个角度审视自己…",
```

File: src/main/resources/assets/ltr/lang/zh_cn.json [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+    "text.ltr.meditation7": "想象一下，你正漂浮在一片纯粹的祥云中。如果云是淡紫色的，吟诵一句平静的话，直到它没有...",
```
Comment (liach) [inline src/main/resources/assets/ltr/lang/zh_cn.json]:
```suggestion
    "text.ltr.meditation7": "想象一下，你正漂浮在一片纯粹的祥云中。如果云是淡紫色的，吟诵一句平静的话，直到它没有…",
```

File: src/main/resources/assets/ltr/lang/zh_cn.json [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+    "text.ltr.meditation8": "放空你的大脑。专注于柱子，专注于驱散不和谐的力量...",
```
Comment (liach) [inline src/main/resources/assets/ltr/lang/zh_cn.json]:
```suggestion
    "text.ltr.meditation8": "放空你的大脑。专注于柱子，专注于驱散不和谐的力量…",
```

File: src/main/resources/assets/ltr/lang/zh_cn.json [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+    "text.ltr.meditation9": "你的超验悟道之旅可能很长，但耐心是你维持精神健康的关键...",
```
Comment (liach) [inline src/main/resources/assets/ltr/lang/zh_cn.json]:
```suggestion
    "text.ltr.meditation9": "你的超验悟道之旅可能很长，但耐心是你维持精神健康的关键…",
```

**Comment by qsefthuopq** [comment]
It wasn't necessary to change some Chinese full Angle symbols because they occupy different space and look ugly.

**Pull request merged**